pkg/rbeconfigsgen/rbeconfigsgen.go

```diff
@@ -379,7 +379,7 @@ func installBazelisk(d *dockerRunner, downloadDir, execOS string) (string, error
 	}
 	defer resp.Body.Close()
 
-	localPath := path.Join(downloadDir, filename)
+	localPath := filepath.Join(downloadDir, filename)
```

Comment:
Curious: What errors do you see when using `path.Join` on Windows?

Reply:
Nvm - `PS > cmd /r /unix/style/path` indeed doesn't work. I get errors eg `Invalid Switch - "Users"`. 

Running it directly from `cmd` seems to work, but IIUC the container could be using either cmd or PS.

I tweaked it so it only cleans uses windows filepath on the `dir` command so it doesn't potentially break eg bazel paths.